pkg/exchange/okex/stream.go

```diff
@@ -338,3 +331,14 @@ func (s *Stream) dispatchEvent(e interface{}) {
 
 	}
 }
+
+func (s *Stream) getInstrumentType() okexapi.InstrumentType {
+	e := s.exchange.(*Exchange)
```

Comment:
you can use the original type *Exchange of the field exchange here so you don't need to cast the type